interactive_multiselect_printer.go

```diff
@@ -25,9 +26,18 @@ var (
 		Filter:         true,
 		KeySelect:      keys.Enter,
 		KeyConfirm:     keys.Tab,
+		Checkmarks: Checkmarks{
+			Selected:    "✓",
+			NotSelected: "✗",
+		},
 	}
 )
 
+type Checkmarks struct {
+	Selected    string
+	NotSelected string
+}
+
```

Comment:
Could you please rename this to `Checkmark` (singular) and put it into the file `/atoms.go`, it's dedicated to those small helper structs :)